logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/SendConfirmationUseCase.kt

```diff
@@ -0,0 +1,61 @@
+package com.wire.kalium.logic.feature.message
+
+import com.benasher44.uuid.uuid4
+import com.wire.kalium.logic.CoreFailure
+import com.wire.kalium.logic.data.client.ClientRepository
+import com.wire.kalium.logic.data.id.ConversationId
+import com.wire.kalium.logic.data.message.Message
+import com.wire.kalium.logic.data.message.MessageContent
+import com.wire.kalium.logic.data.sync.SlowSyncRepository
+import com.wire.kalium.logic.data.sync.SlowSyncStatus
+import com.wire.kalium.logic.data.user.UserRepository
+import com.wire.kalium.logic.functional.Either
+import com.wire.kalium.logic.functional.flatMap
+import com.wire.kalium.logic.functional.onFailure
+import com.wire.kalium.logic.kaliumLogger
+import kotlinx.coroutines.flow.first
+import kotlinx.datetime.Clock
+
+class SendConfirmationUseCase internal constructor(
```

Comment:
I'd expect the use case to work differently.  I'd say this `SendConfirmationUseCase` an internal use case or maybe it's not a use case at all but an internal helper component.

The read confirmation are sent when the user "reads" messages in a conversation so it probably makes sense to call the `SendConfirmationUseCase` from the `UpdateConversationReadDateUseCase`, which is invoked when the user "reads" messages. 

Delivery confirmations is sent upon receiving messages which is tied to sync inside Kalium. Here I would expect delivery confirmations to be triggered from the `ConversationEventReceiver`.

Lastly I'd like the `SendConfirmationUseCase` to support the feature of multiple confirmations at once so that they can be batched into a single message.

Reply:
Thanks for your review. It is touching the same points that i have about my PR. It is not a method that should be used by a client implementation but it is necessary for QA to test client behavior. We recently found a better place for such things and i moved the use case into the debug scope.

Additionally i implemented the possibility to send multiple confirmations at once. Which i forgot in my PR.

Please re-review 😃